src/views/Editor.vue

```diff
@@ -123,7 +123,12 @@
 			<b-row no-gutters>
 				<b-col md="3" v-if="!!selectedSchema">
 					<b-nav class="sticky-top editor-index-navigation" vertical>
-						<b-nav-item v-for="tab in tabs" :key="tab.uri" :to="`/dataset/${id}/${tab.uri}`">
+						<b-nav-item v-for="(tab, index) in tabs" :key="tab.uri" :to="`/dataset/${id}/${tab.uri}`">
+							<b-badge
+								:variant="$route.params.tab === tab.uri ? 'info' : 'secondary'">
+								{{ index+1 }}
```

Comment:
is it so that tab numbers will be visible in front of tab name? like 
1 Content Description
2 Actors
etc?

Reply:
yes